src/components/cc-doc-list/cc-doc-list.types.d.ts

```diff
@@ -1,6 +1,16 @@
-interface Documentation {
-  desc: string;
-  icons: string[];
-  link: string;
-  title: string;
+import{ DocCardStateLoaded } from '../cc-doc-card/cc-doc-card.types.js';
+
+export type DocListState = DocListStateLoaded | DocListStateLoading | DocListStateError;
+
+export interface DocListStateLoaded {
+  type: 'loaded';
+  docs: DocCardStateLoaded[];
```

Comment:
thought: I think seeing this makes me feel the same way as `cc-article-*`, it feels weird for this case to have a state in another state. :thinking: